scripts/exgdas_atmos_gldas.sh

```diff
@@ -0,0 +1,337 @@
+#! /usr/bin/env bash
+
+################################################################################
+####  UNIX Script Documentation Block
+#                      .                                             .
+# Script name:         exgdas_atmos_gldas.sh
+# Script description:  Runs the global land analysis                    
+#
+################################################################################
+
+source "${HOMEgfs:?}/ush/preamble.sh"
+
+#################################
+# Set up UTILITIES
+#################################
+export FINDDATE=${FINDDATE:-/apps/ops/prod/nco/core/prod_util.v2.0.13/ush/finddate.sh}
```

Comment:
Why are wcoss2 paths hard-wired here and below?
I guess we can handle that in another PR.
Would you add a comment 
```
#TODO - remove hard-wired WCOSS2 default paths
```

Reply:
Good catch. Those defaults should use module-provided paths to those utilities. I will correct in a subsequent PR. Thanks!